.gitignore

```diff
@@ -1,8 +1,4 @@
-# DS_Store
 .DS_Store
-*/.DS_Store
```

Comment:
I remember that these lines were there b/c there were D_Store files at various places